ext/scheduler/prime/scheduler.go

```diff
@@ -47,8 +47,9 @@ func (s *Scheduler) DeployJobs(ctx context.Context, namespace models.NamespaceSp
 	}
 
 	repo := s.jobRunRepoFac.New()
+	jobDestination := "implement me" // fetch job destination from plugin service
```

Comment:
Make this empty string, other flows also return empty string when no destination